lighthouse-core/audits/byte-efficiency/legacy-javascript.js

```diff
@@ -436,6 +436,18 @@ class LegacyJavascript extends ByteEfficiencyAudit {
       };
       for (const match of matches) {
         const {name, line, column} = match;
+
+        let original;
+        const bundle = bundles.find(bundle => bundle.script.src === url);
+        const entry = bundle && bundle.map.findEntry(line, column);
+        if (entry) {
+          original = {
+            file: entry.sourceURL || '<unmapped>',
```

Comment:
WDYT about letting this property be optional so we can be more flexible in display? 

based on the below it looks like we'll prefer `file` over the URL, and it seems worse to show `<unmapped>` for everything and it'd be better to allow the client to figure out how to handle it